spec/unit/facter/mysql_server_id_spec.rb

```diff
@@ -6,21 +6,21 @@
   }
 
   describe "mysql_server_id" do
-    context "igalic's laptop" do
+    context "developer laptop" do
       before :each do
-        Facter.fact(:macaddress).stubs(:value).returns('3c:97:0e:69:fb:e1')
+        Facter.fact(:ipaddress).stubs(:value).returns('192.168.1.100')
       end
       it do
-        Facter.fact(:mysql_server_id).value.to_s.should == '4116385'
+        Facter.fact(:mysql_server_id).value.to_s.should == '1739024'
       end
     end
 
     context "node with lo only" do
       before :each do
-        Facter.fact(:macaddress).stubs(:value).returns('00:00:00:00:00:00')
+        Facter.fact(:ipaddress_lo).stubs(:value).returns('172.0.0.1)
```

Comment:
looks like you're missing a single quote here, which is causing the unit tests to choke